### PR TITLE
[16.0][FIX] spreadsheet_oca: Avoid to show dynamic cols when no cols selected

### PR DESCRIPTION
**Pull request by CarlosRoca13**
Steps to reproduce the error:

1. Go to a pivot
2. Remove all groups of columns to set only the measure
3. Add the pivot to a spreadsheet with dynamic columns and a number of columns

An error will be thrown

cc @Tecnativa TT50912 

ping @pedrobaeza @chienandalu

### CODE REVIEW

**Review by chienandalu — APPROVED** [review]
Tested :+1:

File: spreadsheet_oca/wizards/spreadsheet_select_row_number.py [hunk 1]
```diff
@@ -10,5 +10,6 @@ class SpreadsheetSelectRowNumber(models.TransientModel):
 
     dynamic_rows = fields.Boolean()
     number_of_rows = fields.Integer()
+    can_be_dynamic_cols = fields.Boolean()
```
Comment (pedrobaeza) [inline spreadsheet_oca/wizards/spreadsheet_select_row_number.py]: The variable name is a bit confusing. Maybe you refer to `can_have_dynamic_cols`?

Comment (CarlosRoca13) [reply spreadsheet_oca/wizards/spreadsheet_select_row_number.py]: Changed!

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge minor

**Pull request merged**